hooks/conan-center.py

```diff
@@ -50,7 +50,9 @@
              "KB-H049": "CMAKE WINDOWS EXPORT ALL SYMBOLS",
              "KB-H050": "DEFAULT SHARED OPTION VALUE",
              "KB-H051": "DEFAULT OPTIONS AS DICTIONARY",
-             "KB-H052": "CONFIG.YML HAS NEW VERSION"
+             "KB-H052": "CONFIG.YML HAS NEW VERSION",
+             "KB-H053": "PRIVATE IMPORTS",
+             "KB-H054": "INVALID SYMLINKS"
```

Comment:
```suggestion
             "KB-H053": "INVALID SYMLINKS"
             "KB-H054": "PRIVATE IMPORTS",
```

Right now it should go in this ordering, unless we wait after the _symlinks_ issue is resolved